### ISSUE TITLE
Incorrect sysconfig configuration with simple install on ubuntu 14.04 LTS

### ISSUE THREAD

**Issue by masha256** [edelight/chef-mongodb#301]
I am using ubuntu 14.04 LTS with the following basic chef/mongo config:

I have the following default attributes set:

```
"mongodb": {
        "package_version": "2.6.1"
    }
```

And have the following runlist for my node(s):

```
    recipe[mongodb::mongodb_org_repo]
    recipe[mongodb::install]
```

Upon cooking the node, mongodb was installed successfully from the 10gen repo, but because of a mongodb::install file override (line 5 of install.rb), ended up with just this in my "/etc/default/mongodb" sysconfig file:

```
ENABLE_MONGODB=no
```

The upstart script that was installed is not functional. It checks for 'ENABLE_MONGOD' (notice there is no 'B' at the end). I see the chef template file defaults/mongodb.sysconfig.erb would have correctly handled this by pulling in the attributes from attributes/sysconfig.rb and set it from one of the variables described in the "these are backward compat purposes", but because the "install" recipe explicitly set the content of this file, the "ENABLE_MONGOD=yes" was never set.

The end result is a mongodb successfully installed, but that doesn't run via "service mongodb start".

Did I misconfigure my chef run list and/or attributes, or is this a bug in the cookbook?

**Comment by jamesonjlee** [comment]
yes, we intentionally disabled the included upstart script so we can modify log dir, db dir, and such.  We later update that properly to enable allowing mongo to run though.

This is on ubuntu 14.04 LTS?

**Comment by josephholsten** [comment]
we should definitely add a kitchen test for ubuntu 14.04.
